src/main/java/uk/gov/hmcts/reform/prl/controllers/C100IssueCaseController.java

```diff
@@ -55,30 +59,41 @@ public AboutToStartOrSubmitCallbackResponse issueAndSendToLocalCourt(
             requireNonNull(caseData);
             sendgridService.sendEmail(c100JsonMapper.map(caseData));
         }
+        caseData = caseData.toBuilder().issueDate(LocalDate.now()).build();
         Map<String, Object> caseDataUpdated = callbackRequest.getCaseDetails().getData();
 
         if (null != caseData.getCourtList() && null != caseData.getCourtList().getValue()) {
-            String baseLocationId = caseData.getCourtList().getValue().getCode();
+            String[] idEmail = caseData.getCourtList().getValue().getCode().split(":");
+            String baseLocationId = Arrays.stream(idEmail).toArray()[0].toString();
             String key = locationRefDataService.getCourtDetailsFromEpimmsId(baseLocationId, authorisation);
             String[] venueDetails = key.split("-");
             String regionId = Arrays.stream(venueDetails).toArray()[1].toString();
             String courtName = Arrays.stream(venueDetails).toArray()[2].toString();
             String regionName = Arrays.stream(venueDetails).toArray()[4].toString();
             String baseLocationName = Arrays.stream(venueDetails).toArray()[5].toString();
```

Comment:
move it to a static utility method. use the manage order base branch for this. Speak to Sonali